haste/windowed_conformal_model/time_series_features.py

```diff
@@ -14,6 +14,8 @@
 import pandas as pd
 from pygam import LinearGAM
 
+
+# TODO: reorder params, timestamps first
 def time_series_features(feature_time_series, timestamps, end_time):
```

Comment:
OK, so we compute the ts-features on the whole well-substream, in which case, what are the values for `all_features`, `features_for_new_image` we pass into here?

Reply:
This function returns the features_for_new_image. The function will have already been used offline to compute all_features for the "training" data. These are then fed into the interestingness function